tests/1-login.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+
+const user = {
+  name: 'Jane Doe',
+  email: 'customer@practicesoftwaretesting.com',
+  password: 'welcome01'
+}
+
+test('login', async ({ page }) => {
+  await page.goto('https://practicesoftwaretesting.com/auth/login');
+  await page.locator('#email').fill(user.email);
+  await page.locator('#password').fill(user.password);
+  await page.locator('input.btnSubmit').click();
+  await expect(page).toHaveURL('https://practicesoftwaretesting.com/account');
+  // line 11 i think more reliable
+  // await expect(page.getByText('My account')).toBeVisible();
+  await expect(page.locator('[data-test="page-title"]')).toContainText('My account');
```

Comment:
Please consider this option in config file https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute in order to use getByTestId() method